<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>normalization_dialog</class>
 <widget class="QDialog" name="normalization_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>211</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>211</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Normalization</string>
  </property>
  <widget class="QFrame" name="normalizationFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>211</width>
     <height>140</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Exact Electroncount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculated Electroncount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="exactCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="normalizationButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>190</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Normalization</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="calculatedCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
